docs/code/code-repo-management.asciidoc

```diff
@@ -7,17 +7,35 @@ image::images/code-repo-management.png[]
 
 [float]
 ==== Add and delete a repo
-The <<code-import-first-repo, Import your first repository>> page provides step-by-step instructions for adding a GitHub repo to *Code*.  You can fine tune the hostname of the git clone URL in your `kibana.yml` file.
+The <<code-import-first-repo, Import your first repository>> page provides step-by-step instructions for adding a GitHub repo to *Code*.  You can fine-tune the hostname of the git clone URL in your `kibana.yml` file.
 
-For security reasons, Code allows only a few trusted hostnames, such as github.com, by default. You can add an SSH key to {kib} to clone private repos.
+For security reasons, Code allows only a few <<#clone-url-management, trusted hostnames>>, such as github.com. To clone private repositories <<#clone-private-repo-with-ssh-key, add an SSH key>>.
 
-Deleting a repo removes it from local storage and the Elasticsearch index.
+To delete a repository, find the go to the **Repositories** tab, find the name of the repo and click *Delete*.
+
+[float]
+==== Clone private repo with SSH key
+Clones of private repos require an SSH key for authentication. The username associated with your host must have write access to the repository you want to clone.
+
+The following section provides links for generating your ssh key through GitHub. If you already have an SSH key added through your host, skip to step 4.
+
+1. https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key[Generate an ssh key].
+
+2. https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent[Add the ssh key to your ssh-agent.]
+
+3. https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account[Add the ssh key to your host].
+
+4. Copy the ssh key into `data/code/credentials/` under the {kib} folder.
+
+You can now copy private Git repositories into Code.
+
+To delete a repository, find the go to the **Repositories** tab, find the name of the repo and click *Delete*.
 
 [float]
 ==== Reindex a repo
-*Code* automatically reindexes an imported repo at set intervals, but in some cases you might need to manually refresh the index. For example, you might refresh an index  after a new language server is installed. Or, you might want to immediately update the index to the HEAD revision. Click *Reindex* to initiate a reindex.
+*Code* automatically reindexes an imported repo at set intervals, but in some cases, you might need to refresh the index manually. For example, you might refresh an index after a new language server installs. Or, you might want to update the index to the HEAD revision immediately. Click *Reindex* to initiate a reindex.
```

Comment:
Maybe you can find a way to pull out "Click Reindex to initiate a reindex" so you don't have to use it in both paragraphs.